### DISCUSSION TITLE
Intercepting traffic between router and companion Andriod app DGA4231VDF (VCNT-I) - Vodafone UK

### DISCUSSION THREAD

**Discussion by dobypog** [hack-technicolor/hack-technicolor#176] [General]
This may be a bit off-topic to the majority of issues here, but it is related to hacking Technicolor routers. 

I'm trying to intercept the traffic between the Android Vodafone companion app (eu.reply.cordless.uk) and a DGA4231VDF (VCNT-I) (THG3000) Vodafone UK router. 

The app primarily communicates with the router's nginx webserver on TCP port 6699, but uses 6698 for pairing, port 8888 for ubus and port 80 for...I don't know but is the browser interface. 

As a side note, starting in app version 4.4.1 certificate pinning was introduced which I currently can't bypass even after trying all the usual bypass solutions, but the issue I currently face is unrelated to this and I'm using version 4.4.0 so it isn't a problem. 

Using Kali Linux to arpspoof the Andriod device, and forwarding the relevant ports to a BurpSuite transparent listen port, I can't get the nginx server to reply to API calls. The TLS handshake is completed successfully (self signed CA installed on machine and rooted Android device) but any POST requests generated by the app to port 6699 result in 403 Forbidden (same with port 443). If I repeat the same call to port 6698 I get a 200 OK response, but the request isn't understood and results in an error. 

Can anyone with any knowledge in this area suggest why the nginx server replies with 403 Forbidden? Authentication?

**Comment by mbana** [comment]
Why was the issue closed? Should also discussion take place here now?

Anyway, does anyone have an update on this?
